app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerClassifiedBanner.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ *
+ */
+
+package com.wire.android.ui.home.messagecomposer
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import com.wire.android.ui.common.SecurityClassificationBanner
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.spacers.VerticalSpace
+import com.wire.kalium.logic.feature.conversation.SecurityClassificationType
+
+@Composable
+fun MessageComposerClassifiedBanner(
+    securityClassificationType: SecurityClassificationType,
+    paddingValues: PaddingValues = PaddingValues()
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(color = colorsScheme().backgroundVariant)
+            .padding(paddingValues)
+    ) {
+        val isClassifiedConversation = securityClassificationType != SecurityClassificationType.NONE
```

Comment:
If the conversation classification type is set to `NONE` the `Row` element will still be there right? even though it doesn't have any height or anything that should be shown, maybe we could "remove" it completely if there is no need to show based on the classification type? let me know what you think

Reply:
good eye! got catch while refactoring this component

Reply:
I just moved the if block outside